modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/logic/UIItemsBuilder.java

```diff
@@ -576,10 +576,7 @@ public DropdownItem createDeleteVersionDropdownItem() {
 			"deleteURL",
 			PortletURLBuilder.create(
 				_getActionURL(
-					"/document_library/edit_file_entry", Constants.DELETE,
-					_getRenderURL(
-						"/document_library/view_file_entry", _getCurrentURL()
-					).toString())
+					"/document_library/edit_file_entry", Constants.DELETE)
```

Comment:
@TaiPhamLR You should add an explicit redirect to the current url. Otherwise, these action urls would be reexecuted if the user refresh the page (or navigates back/forth using the browser history).

Reply:
@adolfopa, We were also quite confused at first but when compared with other places using _getActionURL function. I see they use _getActionURL(String mvcActionCommandName, String cmd) and _getCurrentURL will be added at: https://github.com/liferay-lima/liferay-portal/blob/516697c0acc813efd139127b88969b227de4addc/modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/logic/UIItemsBuilder.java#L1089.
So that is the reason we don't add it here. Please let me know your thought. Thanks!

Reply:
Ah, you're right. Sorry.